services/harmony/app/util/variables.ts

```diff
@@ -99,6 +99,37 @@ export function getCoordinateVariables(variables: CmrUmmVariable[]): CmrUmmVaria
   return variables.filter((v) => v.umm.VariableType === coordinateType);
 }
 
+/**
+ * validate variable and queryVars parameters, throw exception if failed
+ * @param variableIds - The variable ids in url path
+ * @param queryVars - The variables in query params
+ */
+function validateVariables(variableIds: string[], queryVars: string | string[]): void {
+  if (variableIds.indexOf('all') !== -1 && variableIds.length !== 1) {
+    throw new RequestValidationError('"all" cannot be specified alongside other variables');
+  }
+
+  if (variableIds.indexOf('all') !== -1 && variableIds.length !== 1) {
```

Comment:
Isn't this the same check as above? Was this intended to check if variables were also being set in `queryVars`?